resources/sugarcrm/accountshandler.cpp

```diff
@@ -164,10 +164,11 @@ SugarAccount AccountsHandler::nameValueListToSugarAccount(const KDSoapGenerated:
     SugarAccount account;
     account.setId(id);
     Q_FOREACH (const KDSoapGenerated::TNS__Name_value &namedValue, valueList.items()) {
-        const QString crmFieldName = sugarFieldToCrmField(namedValue.name());
+        const QString crmFieldName = ModuleHandler::sugarFieldToCrmField(namedValue.name());
         const QString value = KDCRMUtils::decodeXML(namedValue.value());
-        const SugarAccount::AccessorHash::const_iterator accessIt = mAccessors.constFind(crmFieldName);
-        if (accessIt == mAccessors.constEnd()) {
+        SugarAccount::AccessorHash accessors = SugarAccount::accessorHash();
```

Comment:
move it out of the foreach, so that it's done only once
make the variable const